test/result.md

```diff
@@ -0,0 +1,229 @@
+# API Reference
```

Comment:
Is this file supposed to be here?

Reply:
Assuming it was an error, I'm going to delete it and merge this PR which is blocking a lot of projects.